types/cancan/cancan-tests.ts

```diff
@@ -0,0 +1,32 @@
+import * as CanCan from 'cancan';
```

Comment:
Since `CanCan` is class, it should be `import CanCan =  require('cancan');` here